frontend/public/index.html

```diff
@@ -2,13 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/logomark.ico" />
```

Comment:
can we continue to call it favicon? that's the normal name for... well, the favicon.